### ISSUE TITLE
Import parsers from WebOfTrust/keriox

### ISSUE THREAD

**Issue by 3N4N** [WebOfTrust/cesrox#22]
Although the deserializers (implementation of `FromStr` trait) were imported from WebOfTrust/keriox in commit 07da651260f9, none of the parsers were.  The parsers are necessary for parsing data streams, and as I understand, they should be included in cesrox.

The parsers in WebOfTrust/keriox can be found in `event_parsing` module.  You can grep for "nom::IResult" to take a quick look at which parsers are implemented in keriox.

**Comment by kentbull** [comment]
Yes, there is still a significant featureset to pull over from Keriox, the parsers being a major part of it. That is one of the next major components to bring in.
